ddpui/tests/core/test_celery_tasks.py

```
@@ -652,3 +662,255 @@ def test_get_connection_catalog_task_success(org_without_workspace: Org):
+    result = clear_stuck_locks()
+
+    assert result == 0
+    assert TaskLock.objects.count() == 0
+
+
+def test_clear_stuck_locks_flow_run_not_found(orguser):
+    """Test clear_stuck_locks when flow run is not found in Prefect"""
+    orgtask = create_test_orgtask(orguser)
+
+    # Create a lock with a flow_run_id
+    lock = TaskLock.objects.create(
```

Ruff (0.15.2)

[error] Local variable `lock` is assigned to but never used

Remove assignment to unused variable `lock`

(F841)